mne/io/brainvision/tests/data/test_segmentation_impedance.vhdr

```diff
@@ -0,0 +1,143 @@
+Brain Vision Data Exchange Header File Version 1.0
+; Data created by the Vision Recorder
+
+[Common Infos]
+Codepage=UTF-8
+DataFile=test.eeg
+MarkerFile=test.vmrk
+DataFormat=BINARY
+; Data orientation: MULTIPLEXED=ch1,pt1, ch2,pt1 ...
+DataOrientation=MULTIPLEXED
+NumberOfChannels=19
+; Sampling interval in microseconds
+SamplingInterval=1000
+SegmentationType=MARKERBASED
+SegmentDataPoints=1100
+Averaged=YES
+AveragedSegments=80
```

Comment:
does it mean it should be read as an Evoked dataset?

Reply:
Yes, if Averaged is set to YES, it means that the data file contains data that have been averaged across segments.

Reply:
this tells me that we should add a read_evoked_brainvision no?
I would rather not have raw_raw_brainvision read such a header without complaining

Reply:
Probably? I'm not really sure as i do not know the inner workings of MNE that well.